applications/minotari_node/src/grpc/base_node_grpc_server.rs

```diff
@@ -426,7 +426,7 @@ impl tari_rpc::base_node_server::BaseNode for BaseNodeGrpcServer {
                         obscure_error_if_true(report_error_flag, Status::internal(e.to_string()))
                     })?;
                 let target_time = constants.pow_target_block_interval(PowAlgorithm::Sha3x);
-                let estimated_hash_rate = target_difficulty.as_u64() / target_time;
+                let estimated_hash_rate = target_difficulty.as_u64().saturating_sub(target_time);
```

Comment:
I don't understand... Why do you subtract time from difficulty? Shouldn't you do a division here?

Reply:
you are right, it should be...